### PR TITLE
INC18039XXXX_Larry's Update > master

### PR DESCRIPTION
**Pull request by larrybaugh**
Sup Peeps...my code update is perfect.

### CODE REVIEW

**Comment by jpostenwapa** [comment]
decent start!  not yet perfect

**Review by ndhillon101 — COMMENTED** [review]
Looks good

**Comment by bvetter95** [comment]
looks great

**Review by bvetter95 — APPROVED** [review]

File: _slides/11-larrybaugh.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Dogs > Cats
```
Comment (larrybaugh) [inline _slides/11-larrybaugh.md]:
```suggestion
Dogs are much better than Cast!!
```

**Pull request merged**